app/src/forms/auth/service.js

```diff
@@ -129,8 +130,10 @@ const service = {
       // what if name or email changed?
       user = await service.updateUser(user.id, obj);
     }
+
+    const usernameIdp = user.idpCode ? `${user.username}@${user.idpCode}` : user.username;
     // return with the db id...
-    return { id: user.id, ...userInfo };
+    return { id: user.id, usernameIdp: usernameIdp,  ...userInfo };
```

Comment:
Now that I look at this, may not even need to do that const declaration and in-line the ternary directly as an option:
```js
return {
  id: user.id,
  usernameIdp: user.idpCode ? `${user.username}@${user.idpCode}` : user.username,
  ...userInfo
};
```

Reply:
okay, will change this to be an inline ternary value